openassessment/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Celery needs to be loaded so that task registration and discovery can work correctly.
+"""
+# This will make sure the app is always imported when Django starts so that shared_task will use this app.
+from .celery import APP as CELERY_APP
```

Comment:
Don't we want to import this only in `settings.test`? When run as part of edxapp, the settings defined in edx-platform should apply.